Generated rust bindings syntax error when deserializing ABI

**Version**
Bindings generated with forge using
ethers-rs#b2fc9fdf50d6fe3e81de0ac5648a068425cf87a7

My repo ethers version
```
└── ethers v0.17.0 (https://github.com/gakonst/ethers-rs#454449b0)
    ├── ethers-addressbook v0.17.0 (https://github.com/gakonst/ethers-rs#454449b0)
    │   ├── ethers-core v0.17.0 (https://github.com/gakonst/ethers-rs#454449b0)
    ├── ethers-contract v0.17.0 (https://github.com/gakonst/ethers-rs#454449b0)
    │   ├── ethers-contract-abigen v0.17.0 (https://github.com/gakonst/ethers-rs#454449b0)
    │   │   ├── ethers-core v0.17.0 (https://github.com/gakonst/ethers-rs#454449b0) (*)
    │   ├── ethers-contract-derive v0.17.0 (proc-macro) (https://github.com/gakonst/ethers-rs#454449b0)
    │   │   ├── ethers-contract-abigen v0.17.0 (https://github.com/gakonst/ethers-rs#454449b0) (*)
    │   │   ├── ethers-core v0.17.0 (https://github.com/gakonst/ethers-rs#454449b0) (*)
    │   ├── ethers-core v0.17.0 (https://github.com/gakonst/ethers-rs#454449b0) (*)
    │   ├── ethers-providers v0.17.0 (https://github.com/gakonst/ethers-rs#454449b0)
    │   │   ├── ethers-core v0.17.0 (https://github.com/gakonst/ethers-rs#454449b0) (*)
    ├── ethers-core v0.17.0 (https://github.com/gakonst/ethers-rs#454449b0) (*)
    ├── ethers-etherscan v0.17.0 (https://github.com/gakonst/ethers-rs#454449b0)
    │   ├── ethers-core v0.17.0 (https://github.com/gakonst/ethers-rs#454449b0) (*)
    ├── ethers-middleware v0.17.0 (https://github.com/gakonst/ethers-rs#454449b0)
    │   ├── ethers-contract v0.17.0 (https://github.com/gakonst/ethers-rs#454449b0) (*)
    │   ├── ethers-core v0.17.0 (https://github.com/gakonst/ethers-rs#454449b0) (*)
    │   ├── ethers-etherscan v0.17.0 (https://github.com/gakonst/ethers-rs#454449b0) (*)
    │   ├── ethers-providers v0.17.0 (https://github.com/gakonst/ethers-rs#454449b0) (*)
    │   ├── ethers-signers v0.17.0 (https://github.com/gakonst/ethers-rs#454449b0)
    │   │   ├── ethers-core v0.17.0 (https://github.com/gakonst/ethers-rs#454449b0) (*)
    ├── ethers-providers v0.17.0 (https://github.com/gakonst/ethers-rs#454449b0) (*)
    ├── ethers-signers v0.17.0 (https://github.com/gakonst/ethers-rs#454449b0) (*)
    └── ethers-solc v0.17.0 (https://github.com/gakonst/ethers-rs#454449b0)
        ├── ethers-core v0.17.0 (https://github.com/gakonst/ethers-rs#454449b0) (*)
├── ethers v0.17.0 (https://github.com/gakonst/ethers-rs#454449b0) (*)

```

**Platform**
Darwin Kernel Version 21.6.0: arm64

**Description**
Rust bindings generated with foundry have syntax errors.
I successfully created bindings for my contracts 3 weeks ago. The contracts have not changed and the new bindings generated have syntax errors.

There are a few different types of errors, but likely all resulting from the same cause (a missing quotation or back-slash).
e.g.

```
error: unknown start of token: \
   --> bindings/src/playground.rs:163:30
    |
163 |                  "parityPoint\",\"outputs\":[{\"internalType\":\"uint8\",\"name\":\"\",\"type\":\"\

```
```
error: suffixes on a string literal are invalid
  --> bindings/src/ownable.rs:22:17
   |
22 | /                 "[{\"inputs\":[{\"internalType\":\"address\",\"name\":\"previousOwner\",\"type\":\\
23 | |                  "address\",\"components\":[],\"indexed\":true},{\"internalType\":\"address\",\"\
   | |_________________________^ invalid suffix `address`

error: expected one of `)`, `,`, `.`, `?`, or an operator, found `",\"components\":[],\"indexed\":true},{\"internalType\":\"address\",\"\
...
```

```
error[E0061]: this function takes 1 argument but 2 arguments were supplied
    --> bindings/src/ownable.rs:21:13
     |
21   |               ethers::core::utils::__serde_json::from_str(
     |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
22   |                   "[{\"inputs\":[{\"internalType\":\"address\",\"name\":\"previousOwner\",\"type\":\\
23   |                    "address\",\"components\":[],\"indexed\":true},{\"internalType\":\"address\",\"\
     |  ___________________________-
24   | |                  name\":\"newOwner\",\"type\":\"address\",\"components\":[],\"indexed\":true}],\"\
25   | |                  type\":\"event\",\"name\":\"OwnershipTransferred\",\"outputs\":[],\"anonymous\":\
26   | |                  false},{\"inputs\":[],\"stateMutability\":\"view\",\"type\":\"function\",\"name\"\
...    |
31   | |                  components\":[]}],\"stateMutability\":\"nonpayable\",\"type\":\"function\",\"\
32   | |                  name\":\"transferOwnership\",\"outputs\":[]}]",
     | |_______________________________________________________________- argument of type `&'static str` unexpected
     |
note: function defined here
    --> /Users/jonathaneasterman/.cargo/registry/src/github.com-1ecc6299db9ec823/serde_json-1.0.85/src/de.rs:2609:8
     |
2609 | pub fn from_str<'a, T>(s: &'a str) -> Result<T>
     |        ^^^^^^^^
help: remove the extra argument
     |
21   ~             ethers::core::utils::__serde_json::from_str("[{\"inputs\":[{\"internalType\":\"address\",\"name\":\"previousOwner\",\"type\":\\
22   +                  "address)
     |
```

--- a/ethers-contract/ethers-contract-abigen/src/contract/common.rs
+++ b/ethers-contract/ethers-contract-abigen/src/contract/common.rs
@@ -99,11 +99,16 @@ pub(crate) fn struct_declaration(cx: &Context) -> TokenStream {
 
     let abi_parse = if !cx.human_readable {
         quote! {
-            pub static #abi_name: #ethers_contract::Lazy<#ethers_core::abi::Abi> = #ethers_contract::Lazy::new(|| #ethers_core::utils::__serde_json::from_str(#abi)
+            #[rustfmt::skip]
+            const __ABI: &str = #abi;
+
+            /// The parsed JSON-ABI of the contract.
+            pub static #abi_name: #ethers_contract::Lazy<#ethers_core::abi::Abi> = #ethers_contract::Lazy::new(|| #ethers_core::utils::__serde_json::from_str(__ABI)
                                               .expect("invalid abi"));
         }
     } else {
         quote! {
+            /// The parsed human readable ABI of the contract.
             pub static #abi_name: #ethers_contract::Lazy<#ethers_core::abi::Abi> = #ethers_contract::Lazy::new(|| #ethers_core::abi::parse_abi_str(#abi)
                                                 .expect("invalid abi"));
         }

